EB and Test Report Updates

tcole3 opened the pull request:
Updated EB Annotations and Test Result json
Updated all.html test report to reflect EB updates
Updated less-than-2.html to reflect EB updates
Fixed sub-test id attribute values and associated internal links
Updated bulleted list in header per WG call discussion

tcole3 committed:
EB and Test Report Updates

Updated EB Annotations and Test Result json
Updated all.html test report to reflect EB updates
Updated less-than-2.html to reflect EB updates
Fixed sub-test id attribute values and associated internal links
Updated bulleted list in header per WG call discussion

halindrome commented:
I am trying to figure out what you did here - did you manually edit the report file?  If so, what were you trying to accomplish?

tcole3 commented:
Yes, manually updated test summary reports all.html, complete-fails.html and less-than-2.html, as well as report-header.html. (This PR also updates all the annotations and individual test result files associated with the EB implementation to latest version of the implementation.) Main objective was to avoid having to rerun and remerge everything for EB updates; also to eliminate an artifact associated with an outdated TK test result file (which presumably went away over the weekend), and lastly to correct errors in the id attribute values in all.html. Also, addressed issue found through manual double-checking that uncovered a sub-test that was too lax and passed for a particular feature 2 implementations that it shouldn't have: 
1. EB implementation was updated to implement an additional of the exit criteria features. /annotation-model/anno*-manual.html tests were re-run for each of the 3 EB annotations. Per discussion Friday, the resulting changes in the EB column of all.html, etc., were updated manually to avoid you having to remerge and EB*.anno files and EB*.json test result files were updated. 
2. eliminated row in all.html that was artifact of including test results for TK annotation done prior to last changes made in test sequence to accommodate CR3. Assuming you've now re-run TK annotation, wouldn't have to do this.
3. Made updates in the header format, etc. per discussions Friday. Made changes not only in report-header.html, but also made corresponding changes in all.html, complete-fails.html and less-than-2.html AND at same time fixed the id attribute values in all.html for non-must tests so they could be properly referenced from table in header. Compare to https://tcole3.github.io/test-results/annotation-model/all.html to see changes in content and internal linking.
4. Also corrected all.html, reducing from 8 to 6 the number of implementations passing the Embedded Textual Body tests - appears that this test is too lax since it identified 2 implementations as using Textual Bodies that did not.
